sentences/de/binary_sensor_HassGetState.yaml

```
@@ -132,3 +132,42 @@ intents:
         slots:
           domain: binary_sensor
           device_class: carbon_monoxide
+
+      # Connectivity
+      - sentences:
+          - "(ist|sind) <name> [<area>] {bs_connectivity_states:state}"
+        response: einzeln_janein
+        requires_context:
+          domain: binary_sensor
+          device_class: connectivity
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "(ist|sind) [<irgend> [der]] <ding> [<area>] {bs_connectivity_states:state}"
+        response: irgendeins
+        slots:
+          domain: binary_sensor
+          device_class: connectivity
+
+      - sentences:
+          - "sind alle <ding> [<area>] {bs_connectivity_states:state}"
```

```suggestion
          - "sind alle <ding> [<area>] {bs_connectivity_states:state}"
          - "sind alle <ding> {bs_connectivity_states:state} <area>"
          - "sind <area> alle <ding> {bs_connectivity_states:state}"
```

_4 possible test sentences_